contrib/terraform/upcloud/README.md

```diff
@@ -108,6 +110,8 @@ terraform destroy --var-file cluster-settings.tfvars \
   * `cpu`: number of cpu cores
   * `mem`: memory size in MB
   * `disk_size`: The size of the storage in GB
+  * `force_public_ip`: If `use_public_ips` is set to false, this forces a public NIC on the machine. Useful if you're migrating from public nodes to only private. Defaults to `false`
+  * `force_no_user_data`: If `private_network_dns` is set existing nodes will be deleted. This forces this mahcine to not add the user_data. Useful if you're migrating from public nodes to only private. Defaults to `false`
```

Comment:
Should there be some migration steps to move from public to private and vice verse?

Reply:
Hmm, yea that makes sense. I'll write some small section about that. But it's basically to just replace all nodes.

Reply:
When trying to apply these changes on an existing cluster I'm seeing `will be updated in-place`, is this not true or what do you mean by "replace all nodes"?

Reply:
The issue is that the indexing for the interfaces in terraform matter. So if you remove the public IP it will look like it's in place. But in reality it will recreate the interface completely, so it will get a new IP. See [this issue](https://github.com/UpCloudLtd/terraform-provider-upcloud/issues/650)

Reply:
That's scary... :o